compiler/compile/OMRCompilation.cpp

```diff
@@ -1293,36 +1293,29 @@ bool OMR::Compilation::incInlineDepth(TR::ResolvedMethodSymbol * method, TR::Nod
    int32_t cpIndex = callSymRef->getCPIndex();
    TR_ByteCodeInfo &bcInfo = callNode->getByteCodeInfo();
 
+   TR_AOTMethodInfo *aotMethodInfo = NULL;
    if (self()->compileRelocatableCode())
       {
-      TR_AOTMethodInfo *aotMethodInfo = (TR_AOTMethodInfo *)self()->trMemory()->allocateHeapMemory(sizeof(TR_AOTMethodInfo));
+      aotMethodInfo = (TR_AOTMethodInfo *)self()->trMemory()->allocateHeapMemory(sizeof(TR_AOTMethodInfo));
```

Comment:
Let's use C++ style casts wherever possible. They have the advantage of getting compiler checked as well as easier searchability. Similarly elsewhere.

Reply:
Done.